.github/workflows/publish-and-deploy-gamma.yaml

```diff
@@ -0,0 +1,27 @@
+name: Publish and Deploy (Gamma)
+permissions: read-all
+
+on:
+  push:
+    branches: [ dev ]
+  workflow_dispatch:
+    branches:
+      - dev
```

Comment:
```suggestion
```

Reply:
why?

Reply:
https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow

Reply:
Never mind, it says `in`